### ISSUE TITLE
Change the sentence's text color when "License:" is set to "Licensing issue".

### ISSUE THREAD

**Issue by ckjpn** [Tatoeba/tatoeba2#2996]
### The Suggested Enhancement

Change the sentence's text color when "License:" is set to "Licensing issue".

I think using the same color that is being used for "Correctness = -1" would be appropriate.

### Notes

* This red color would indicate which sentences are not included in the exported data.
* This red color would immediately show people browsing the data via the website that there is some kind of problem with such sentences, even when shown as translations.
* This red color would let those trying to translate such sentences know that they can't translate them.  The "License:" listing on the page doesn't show to those who are translating while browsing search results and is often overlooked even when members are on the sentence's page. (Related Wall post: https://tatoeba.org/en/wall/show_message/39055#!#message_39055)

### Example of a "licensing issue" sentence.

https://tatoeba.org/en/sentences/show/2392255

**Comment by biancadene** [comment]
Is this still available?

**Comment by jiru** [comment]
@biancadene Yes. By the way, the linked sentence [2392255](https://tatoeba.org/en/sentences/show/2392255) has been set as "incorrect" so that it now shows up in red, but setting the license to "licensing issue" alone still does not turn a sentence into red.

**Comment by ckjpn** [comment]
Eventually, tatoeba.org should hide these sentences, since leaving them exposed on a publicly-available webpage means tatoeba.org is "distributing" copyrighted material.

Priority, would be to hide these pages from any non-logged in person.
But, tatoeba.org shouldn't be sharing these with anyone, I would think.

Maybe TRANG needs to be consulted since the site is hers.

**Comment by ckjpn** [comment]
Possibly, you could hide such sentences from everyone except the person who added it and admins, like what's done for spam on the Wall.
